docker/vue-app/src/components/form_editor_view/CategoryItem.js

```diff
@@ -0,0 +1,138 @@
+export default {
+    name: 'category-item',
+    props: {
+        categoriesRecord: Object,
+        availability: String
+    },
+    inject: [
+        'APIroot',
+        'CSRFToken',
+        'libsPath',
+        'categories',
+        'updateCategoriesProperty',
+        'stripAndDecodeHTML',
+        'allStapledFormCatIDs',
+    ],
+    computed: {
+        workflowID() {
+            return parseInt(this.categoriesRecord.workflowID);
+        },
+        cardLibraryClasses() {  //NOTE:? often null (LIVE).  called when smarty referFormLibraryID != ''
+            return `formPreview formLibraryID_${this.categoriesRecord.formLibraryID}`
+        },
+        catID() {
+            return this.categoriesRecord.categoryID;
+        },
+        stapledForms() {
+            let stapledForms = [];
+            this.categoriesRecord.stapledFormIDs.forEach(id => stapledForms.push({...this.categories[id]}));
+            stapledForms = stapledForms.sort((eleA, eleB) => eleA.sort - eleB.sort );
+            return stapledForms;
+        },
+        isStapledToOtherForm() {
+            return this.allStapledFormCatIDs.includes(this.categoriesRecord.categoryID);
+        },
+        /**
+         * NOTE: uses LEAF XSSHelpers.js
+         * @returns {string} truncated category name for card title
+         */
+        categoryName() {
+            let name = this.categoriesRecord.categoryName === '' ? 

```

Comment:
any reason this variable needs to be instantiated if it's just getting returned?

Reply:
It was just a little easier to read and update if it turned out anything further needed to be done with it.  It might have initially had a truncation too, but doesn't after the switch to tables, so it is simple enough to just return at this point.